apps/examples/testcase/le_tc/network/tc_net_netif_core.c

```diff
@@ -0,0 +1,1378 @@
+
+}
+
+/**
+* @testcase				: tc_net_pbuf_strstr_n
+* @brief				:
+* @scenario				:
+* @apicovered			: pbuf_strstr()
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_pbuf_strstr_n(void)
+{
+	u16_t ret;
+	struct pbuf *p = NULL;
+	const char *substr = NULL;
+
+	ret = pbuf_strstr(p, substr);
+
+	TC_ASSERT_EQ("core", ret, 0xffff);
+	TC_SUCCESS_RESULT();
+
+}
+
+/**
+* @testcase				: tc_net_sys_untimeout_n
+* @brief				:
+* @scenario				:
+* @apicovered			: sys_untimeout()
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_sys_untimeout_n(void)
+{
+	sys_timeout_handler handler = NULL;
+	void *arg = NULL;
+
+	sys_untimeout(handler, arg);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase				: tc_net_icmp_dest_unreach_n
+* @brief				:
+* @scenario				:
+* @apicovered			: icmp_dest_unreach()
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_icmp_dest_unreach_n(void)
+{
+	enum icmp_dur_type t;
+	struct pbuf *p = NULL;
+	p = (struct pbuf *)memp_malloc(MEMP_PBUF_POOL);
+	p->payload = NULL;
+	t = 0;
+
+	icmp_dest_unreach(p, t);
+	free(p);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase				: tc_net_icmp_time_exceeded_n
+* @brief				:
+* @scenario				:
+* @apicovered			: icmp_time_exceeded()
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_icmp_time_exceeded_n(void)
+{
+	struct pbuf *p;
+	enum icmp_dur_type t;
+	p = (struct pbuf *)memp_malloc(MEMP_PBUF_POOL);
+
+	p->payload = NULL;
+	t = 0;
+
+	icmp_time_exceeded(p, t);
+	free(p);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase				: tc_net_icmp_dest_unreach_p
+* @brief				:
+* @scenario				:
+* @apicovered			:
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_icmp_dest_unreach_p(void)
+{
+	struct pbuf *p = NULL;
+	enum icmp_dur_type t;
+
+	p = (struct pbuf *)memp_malloc(MEMP_PBUF_POOL);
+	p->payload = "packet";
+	t = 0;
+
+	icmp_dest_unreach(p, t);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase				: tc_net_icmp_time_exceeded_p
+* @brief				:
+* @scenario				:
+* @apicovered			:
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_icmp_time_exceeded_p(void)
+{
+	struct pbuf *p = NULL;
+	enum icmp_dur_type t;
+
+	p = (struct pbuf *)memp_malloc(MEMP_PBUF_POOL);
+	p->payload = "packet";
+	t = 0;
+
+	icmp_time_exceeded(p, t);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase				: tc_net_icmp_send_response_n
+* @brief				:
+* @scenario				:
+* @apicovered			: icmp_send_response()
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_icmp_send_response_n(void)
+{
+	struct pbuf *p = NULL;
+	u8_t type;
+	u8_t code;
+	p = (struct pbuf *)memp_malloc(MEMP_PBUF_POOL);
+	p->payload = NULL;
+	type = 3;
+	code = 0;
+
+	icmp_send_response(p, type, code);
+	free(p);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase				: tc_net_icmp_send_response_p
+* @brief				:
+* @scenario				:
+* @apicovered			: icmp_send_response()
+* @precondition			:
+* @postcondition		:
+*/
+static void tc_net_icmp_send_response_p(void)
+{
+        struct pbuf *p;
```

Comment:
In all above scenarios you did init for pbuf , why you missed here ?
Please maintain code sanity